src/components/todofooter/todofooter.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { useTodos } from '../../context/todoProvider';
+import { deleteTodo } from '../../api/todos';
+import { ErrorType } from '../../types/Error';
+
+export const TodoFooter = () => {
+  const {
+    filterBy, setFilterBy, countIncompleteTask, todos,
+    setTodos, setError, deletingTask, setDeletingTask,
+  } = useTodos();
+
+  const hiddenBtn = todos.filter(el => el.completed).length === 0;
+
+  const DeleteCompletedTask = () => {
+    const completedTodos = todos.filter(task => !task.completed);
```

Comment:
also hide the notification before any next request;

in the start new request, I add setError(null)